### PR TITLE
Fix smoke test compile issue aug 2020

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

File: common/smoke-tests/pom.xml [hunk 1]
```diff
@@ -126,13 +126,13 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.0.3</version>
+      <version>1.1.0</version>
```
Comment (conniey) [inline common/smoke-tests/pom.xml:129]: @danieljurek Should we add Jim's version tags to these so when we release, it doesn't break?

**Review by joshfree — APPROVED** [review]

**Pull request merged**